client/my-sites/upgrades/checkout/secure-payment-form.jsx

```diff
@@ -1,48 +1,56 @@
 /**
  * External dependencies
  */
-var defer = require( 'lodash/defer' ),
-	React = require( 'react' );
+import defer from 'lodash/defer';
```

Comment:
Actually I was confused. I mean it this importing could be similar to ...

```es6
const defer = require( 'lodash').defer;
```

... which isn't a good way to require `defer` lib.